blog.md

```diff
@@ -1,26 +1,78 @@
 ## About
 
-This blog post describes (and is hosted on) a demonstration of a CI/CD pipeline where each Pull Request creates an ephemeral environment that could be reviewed as part of the github review process as well as automatically deploying and upgrading a staging and a production environment.
+Docker and its nascent ecosystem are interesting, but they aren't solving problems -  just moving them around. For me the real problems get solved when we change the discussion from Infrastructure (as a Service) to Platform (as a Service) and the _unit_ delivered by the Platform being a running application rather than a Virtual Machine.
 
-I chose to demonstrate this using _only_ opensource products. This means I am running everything from the _IAAS_ layer up. You could quite easily skip OpenStack and run this in AWS or Digital Ocean.
+Even Kubernetes to me still feels a lot like IaaS, just with containers being the unit of Infrastructure rather than VMs. Of course there are already Platforms out there that focus the user experience towards the application rather than the infrastructure with Cloud Foundry and Heroku being major players in the space.
 
-This is not intended to be a detailed "how to build a *AAS platform" so while I mentioned each layer's underlying technology I will not go into detail until we are at the application itself.
+In the Docker ecosystem we are starting to see more of these types of Platforms, the first of which was [Dokku](https://github.com/progrium/dokku) which started as a single machine [Heroku](http://heroku.com) replacement written in about 100 lines of bash. Building on top of that work other, richer systems like [Deis](http://Deis.io) and [Flynn](http://flynn.io) started to emerge as well as custom solutions built in-house, like Yelp's [PaaSta](https://github.com/Yelp/PaaSta).
+
+Actions speak louder than words so I decided to document (and demonstration) a Platform built from the ground up (using Open Source projects) and then deploying an application to it via a Continuous Integration/Delivery (CI/CD) pipeline.
+
+You could (and probably should) be using a public cloud provider for some and allof this stack, however I wanted to demonstrate that a system like this can be built and run internally as not everybody is able (for many valid reasons) able to use public cloud.
```

Comment:
Some minor things (this is where the markup in Google Doc would be handy):
« You could (and probably **w**ould) **use** a public cloud provider for some **(or all)** of this stack**;** however, I wanted to demonstrate that a system like this can be built and run internally**,** as not everybody is **able to use the public cloud**. »

* Cleaned a couple of typos and repeated words.
* Simplify tense; fix resulting punctuation.
* The reader knows whether the public cloud is right for them or not already. :smile: